### DISCUSSION TITLE
Esc to go to playing song

### DISCUSSION THREAD

**Discussion by madprops** [Taiko2k/Tauon#1195] [Ideas]
This is just something I'm used to doing in another player. When I press Esc in that player, it focuses the current playing song. In Tauon the equivalent is right clicking the play icon. 

1) Is there a kb shortcut for this?

2) Would it be possible to map this to Esc?

The reason I like Esc in particular is because I have it mapped to a side button on my mouse.

**Comment by Solarunit** [comment]
`Esc` is an opposite of obvious shortcut to show playing song. 
You can click on song title at the bottom to show current playing song.
Not sure if there is a shortcut, bit would be nice to have one.

**Comment by Taiko2k** [comment]
Yes there is its currently mapped to the `"` key by default. In the user data folder there will be a copy of the keymap file `input.txt` you can use to change keymaps, what you're looking for in that file is `goto-playing`.

**Reply by madprops** [reply]
Thanks, I easily changed it to Escape and works great.
